src/ImapMailbox.php

```diff
@@ -15,12 +15,14 @@ class ImapMailbox {
 	protected $imapParams = array();
 	protected $serverEncoding;
 	protected $attachmentsDir;
+	protected $attachmentsMaxSize = '';
 
-	public function __construct($imapPath, $login, $password, $attachmentsDir = null, $serverEncoding = 'utf-8') {
+	public function __construct($imapPath, $login, $password, $attachmentsDir = null, $serverEncoding = 'utf-8',$attachmentsMaxSize = '') {
```

Comment:
Too much arguments in __construct. This should be moved to separate setAttachmentMaSize method